Mathlib/Algebra/Ring/CentroidHom.lean

```diff
@@ -467,21 +467,44 @@ lemma centerToCentroid_apply (z : { x // x ∈ NonUnitalSubsemiring.center α })
     (centerToCentroid z) a = z * a := rfl
 
 lemma center_iff_op_centroid (a : α) :
```

Comment:
I don't understand this "op" naming scheme. Can you explain the reasoning behind this name (here and elsewhere)?

Reply:
I think you might be able to just use `mem_center_iff` although the namespace `CentroidHom` is a little weird. Calling this `NonUnitalNonAssocSemiring.mem_center_iff` might make more sense.

Reply:
Thanks, I've made that change.